msbuild/Xamarin.MacDev.Tasks/Tasks/FilterStaticFrameworks.cs

```diff
@@ -0,0 +1,13 @@
+using Xamarin.Messaging.Build.Client;
+
+namespace Xamarin.MacDev.Tasks {
+	public class FilterStaticFrameworks : FilterStaticFrameworksTaskBase {
+		public override bool Execute ()
+		{
+			if (ShouldExecuteRemotely ())
+				return new TaskRunner (SessionId, BuildEngine4).RunAsync (this).Result;
+
+			return base.Execute ();
+		}
```

Comment:
I think this task should implement `ITaskCallback` and `ShouldCreateOutputFile` return false, otherwise it will replace the files already exist on Windows (used as inputs) with empty ones.

Reply:
@emaf fixed